src/api/app/apis/gitea_api/v1/client.rb

```diff
@@ -0,0 +1,69 @@
+module GiteaAPI
```

Comment:
What is `app/apis`?

Reply:
Yeah, maybe a bit to much to create a new folder in `app` for that, I will move it to the `lib` folder